lessons/14-css-more-rules/index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+    <head>
+        <title>CSS Rules</title>
+        <style>
+        .divone {
+            display: inline-block;
+            background-color: blue;
+            padding: 20px;
+            border-right-color: brown;
+            border-right-width: 4px;
+            border-right-style: strong;
```

Comment:
I think you meant to put `solid` here 😄 

https://developer.mozilla.org/en-US/docs/Web/CSS/border-style